Makefile

```diff
@@ -153,6 +153,45 @@ data/full-cohort-data.json: data/cohort-data.json data/random-data.json
 	sed '1d' $(word 2,$^) >> $@
 
 
+### COGS Set Up
+
+# The branch name should be the namespace for the new cohort
+BRANCH := $(shell git branch --show-current)
+
+init-cogs: .cogs
+
+templates/$(BRANCH).tsv:
+	echo -e "Term ID\tLabel\tParent Term\tDefinition\tGECKO Category\tSuggested Categories\tComment\nID\tLABEL\tC % SPLIT=|\tA definition\n\tis-required;" > $@
+
+# required env var GOOGLE_CREDENTIALS
+.cogs: | templates/$(BRANCH).tsv
+	cogs init -u $(EMAIL) -t $(BRANCH)
+	cogs add $< -r 2
+	cogs push
+	cogs open
+
+destroy-cogs: | .cogs
+	cogs delete -f
+
+
+### Validation
+
+build/gecko_labels.tsv: build/gecko.owl src/queries/get_labels.rq | build/robot.jar
+	$(ROBOT) query --input $< --query $(word 2,$^) $@
```

Comment:
`make: *** No rule to make target 'src/queries/get_labels.rq', needed by 'build/gecko_labels.tsv'.  Stop.  `

Reply:
Changed to `robot export`